components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -688,6 +713,41 @@ export default class PluginManagement extends AdminSettings {
         );
     }
 
+    renderRemovePluginModal = ({show, onConfirm, onCancel}) => {
+        const title = (
+            <FormattedMessage
+                id='admin.plugin.remove_modal.title'
+                defaultMessage='Remove plugin?'
+            />
+        );
+
+        const message = (
+            <FormattedMessage
+                id='admin.plugin.remove_modal.desc'
+                defaultMessage='Are you sure you whould like to remove the plugin?'
```

Comment:
```suggestion
                defaultMessage='Are you sure you would like to remove the plugin?'
```
Personally, I think `Are you sure you want to remove the plugin?` sounds more natural. It's semantics, though.